serverlessworkflow/modules/ROOT/nav.adoc

```diff
@@ -1,18 +1,23 @@
-// * xref:index.adoc[Home]
 * xref:release_notes.adoc[Release notes]
 * Getting Started
-** xref:getting-started/create-your-first-workflow-service.adoc[Creating your first workflow service]
-** xref:getting-started/cncf-serverless-workflow-specification-support.adoc[CNCF Serverless Workflow specification]
-** xref:getting-started/getting-familiar-with-our-tooling.adoc[Getting familiar with tooling]
-* Core
-** xref:core/understanding-jq-expressions.adoc[jq expressions]
+// TODO: https://issues.redhat.com/browse/KOGITO-9455
+//** Creating a Workflow Project
+** xref:getting-started/create-your-first-workflow-service.adoc[]
+// We will refactor this section here: https://issues.redhat.com/browse/KOGITO-9451
+//** xref:getting-started/getting-familiar-with-our-tooling.adoc[Getting familiar with tooling]
+* Core Concepts
+** xref:core/cncf-serverless-workflow-specification-support.adoc[]
+** xref:core/handling-events-on-workflows.adoc[Events]
+** xref:core/working-with-callbacks.adoc[Callbacks]
+** xref:core/understanding-jq-expressions.adoc[]
 ** xref:core/understanding-workflow-error-handling.adoc[Error handling]
+** xref:core/configuration-properties.adoc[Configuration]
```

Comment:
```suggestion
** xref:core/configuration-properties.adoc[Configuration Properties]
```

Reply:
Configuration is better and shorter to avoid breaking the left menu.